docs/SettingUp.md

```diff
@@ -54,9 +54,11 @@ If you plan to use Intellij IDEA (highly recommended):
 
    When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [ContactCS’s architecture](DeveloperGuide.md#architecture).
 
+<!---
 1. **Do the tutorials**
    These tutorials will help you get acquainted with the codebase.
 
    * [Tracing code](tutorials/TracingCode.md)
    * [Adding a new command](tutorials/AddRemark.md)
    * [Removing fields](tutorials/RemovingFields.md)
+-->
```

Comment:
commenting these out so that we can easily see what was given to us and figure out if we want to fill those up